streams/fr.m3u

```diff
@@ -78,12 +78,12 @@ http://edge9.iptvnetwork.net/live/france4/playlist.m3u8
 http://edge9.iptvnetwork.net/live/france5/playlist.m3u8
 #EXTINF:-1 tvg-id="France24Arabic.fr" status="online",France 24 Arabic (1080p)
 https://static.france24.com/live/F24_AR_HI_HLS/live_tv.m3u8
-#EXTINF:-1 tvg-id="France24Arabic.fr" status="error",France 24 Arabic (576p)
-https://static.france24.com/live/F24_AR_LO_HLS/live_web.m3u8
```

Comment:
Deleting the "web" version wasn't useful, it still can be used as backup in case the "tv" link goes down.

Reply:
live_tv.m3u8 and live_web.m3u8 are identical. I have added them anyway.